documentation/modules/additional-auth-methods-ostack.adoc

```diff
@@ -0,0 +1,9 @@
+[id="additional-authentication-methods-osp_{context}"]
+= Additional authentication methods for migrations with {osp} source providers
+
+{project-short} versions 2.6 and later support the following authentication methods for migrations with {osp} source providers in addition to the standard username and password credential set:
```

Comment:
Maybe break into two sentences. "...versions 2.6 and later support the following authentication methods for migrations with {osp} source providers. This is in.."